ros/src/computing/planning/motion/packages/waypoint_maker/launch/waypoint_loader.launch

```diff
@@ -1,16 +1,19 @@
 <!-- -->
 <launch>
 	<arg name="multi_lane_csv" default="" />
-    <arg name="decelerate" default="1" />
+	<arg name="decelerate" default="1" />
+	<arg name="use_velocity_visualize" default="false" />
```

Comment:
plz change "visualize" to "visualizer", in whole of your code.

Reply:
Fixed.